templates/Dockerfile-rpm.tmpl

```diff
@@ -105,6 +104,19 @@ RUN [[ %%os%% != centos ]] || [[ %%release%% != 7 ]] || ( \
     yum -y install git && \
     yum -y install  devtoolset-8-gcc-c++ && scl enable devtoolset-8 bash )
 
+# install python 3.8 to be able to execute tools scripts
+ARG PYTHON_VERSION=3.8.16
+RUN yum -y install  bzip2-devel \
+                libffi-devel \
+                xz-devel && \
```

Comment:
An earlier `yum clean all` command cleaned caches and now this command requires to download repo information one more time.

I can see 2 options:
1.  Moving installation of these new packages to earlier in the file, merging them with other package installations.
2. Adding a `yum clean all` somewhere in this `RUN` command to decrease docker image and cache sizes.

Reply:
Moved package installation into upper package installation blocks